site/docs/voice/bxml/startTranscription.mdx

```diff
@@ -0,0 +1,242 @@
+
+## Websocket Packet Format
+
+If a `destination` is specified, it will be sent JSON messages for the duration of the real-time transcription. There will be an initial `start` message when the connection is first established. This will be followed by zero or more `transcription` messages containing transcription updates for the tracks being transcribed. Finally, when a real-time transcription is stopped, a `stop` message will be sent.
+
+### Start and Stop Message Parameters
+
+| Parameter    | Description |
+|:-------------|:------------|
+| eventType    | What type of message this is, one of `start`, or `stop` |
+| metadata     | Details about the real-time transcription this message is for. See further details below. |
+| customParams | (optional) (`start` message only) If any `<CustomParam/>` elements were specified in the `<StartTranscription>` request, they will be copied here as a map of `name : value` pairs |
+
+#### Metadata Parameters
+
+| Parameter                     | Description |
+|:------------------------------|:------------|
+| accountId                     | The user account associated with the call |
+| callId                        | The call id associated with the real-time transcription |
+| realtimetranscriptionId       | The unique id of the real-time transcription |
+| transcriptionName             | The user supplied name of the real-time transcription |
+| tracks                        | A list of one or more tracks being transcribed in real-time |
+| tracks.name                   | The name of the track being transcribed, will be used to identify which transcription updates belong to which track |
+| stabilized                    | Whether transcription updates will be sent only after they have become stable or not |
+
+### Transcription Message Parameters
+
+| Parameter | Description            |
+|:----------|:-----------------------|
+| eventType | Will always be `transcription` |
+| track     | The name of the track this transcription update is for, will be one of the names specified in the `start` message |
+| startTime   | The time at which this segment started |
+| endTime   | The time at which this segment ended |
+| isPartial   | Indicates if the segment is complete |
+| language   | The detected language of the segment |
+| alternatives   | The list of possible transcriptions of this segment |
+| alternatives.transcript   | The transcription of this segment as a flattened string |
+| alternatives.items   | The list of items making up this alternative |
+| alternatives.items.content   | A word or punctuation |
+| alternatives.items.startTime   | The time at which this item started |
+| alternatives.items.endTime   | The time at which this item ended |
+| alternatives.items.confidence   | The confidence score associated with a word or phrase in your transcript. |
+| alternatives.items.stable   | Indicates whether the specified item is stable (true) or if it may change when the segment is complete (false). |
```

Comment:
I think alternatives is redundant or not useful. In AWS RTT I don't even think it's useful at all -- all my tests only give me one alternative, and the confidence is in the items, so how would the user be able to decide which alternative (a set of items) is more reliable? Why would they even use anything other than the highest reliable alternative?
Also, segments are potentially long, so is the list of items. Do customers really want this? Should we add a flag that allows users to decide whether they get this or not if they want to save network?

Reply:
I agree with this

Reply:
`stabilized` is supposed to cut down on network chatter. We could maybe add another flag, but I'm not wild about doing so to be honest. What are you proposing?

Reply:
Removed `alternatives`